### PR TITLE
update 'agg_lt' to updated 'hierarchyUtils::agg' arguments

### PR DESCRIPTION
**Pull request by chacalle**
## Describe changes

Propagate changes from https://github.com/ihmeuw-demographics/hierarchyUtils/pull/47

## What issues are related

Related to https://github.com/ihmeuw-demographics/hierarchyUtils/pull/47

### Packages Repositories

* [X] Have you read the [contributing guidelines](https://github.com/ihmeuw-demographics/packageTemplate/wiki#guide-to-r-package-development) for `ihmeuw-demographics` R packages?
* [X] Have you successfully run `devtools::check()` locally?
* [X] Have you updated or added function (and vignette if applicable) documentation? Did you update the 'man' and 'NAMESPACE' files with `devtools::document()`?
* [X] Have you added in tests for the changes included in the PR?
* [X] Do the changes follow the `ihmeuw-demographics` [code style](https://github.com/ihmeuw-demographics/packageTemplate/wiki/Code-style-guide)?
* [ ] Do the changes need to be immediately included in a new build of [`docker-base`](https://github.com/ihmeuw-demographics/docker-base) or [`docker-internal`](https://github.com/ihmeuw-demographics/docker-internal)? If so follow directions in those repositories to rebuild and redeploy the images.
* [ ] Do the changes require updates to other repositories which use this package? If yes, make the necessary updates in those repos, and consider integration tests for those repositories.

### CODE REVIEW

File: R/agg_lt.R [hunk 1]
```diff
@@ -102,8 +102,8 @@
 #'   dt = dt,
 #'   id_cols = id_cols,
 #'   age_mapping = data.table::data.table(
-#'     age_start = seq(0, 110, 5),
-#'     age_end = c(seq(5, 110, 5), Inf)
+#'     age_start = seq(0, 105, 5),
```
Comment (krpaulson) [inline R/agg_lt.R:105]: Why did you remove the terminal age group?

Comment (chacalle) [reply R/agg_lt.R:107]: Since its already part of `dt` in this example it would require using the `present_agg_severity` argument which I'm avoiding for this example.

File: R/agg_lt.R [hunk 2]
```diff
@@ -172,6 +177,7 @@ agg_lt <- function(dt,
 
   # aggregate qx ------------------------------------------------------------
 
+  message("Aggregating px across age groups")
```
Comment (krpaulson) [inline R/agg_lt.R:180]: If you're going to put a message like this, we should add a `quiet` arg.

**Review by krpaulson — DISMISSED** [review]

File: R/agg_lt.R (hunk 2 above)
Comment (chacalle) [reply R/agg_lt.R:180]: I added messages to `hierarchyUtils` agg and scale too. Since we are using messages people could also use `suppressMessages`, I think we had an issue discussing print vs messages but I can't find it now. 

@spencerpease do you have thoughts on best practices here?
